net/adguardhome/files/adguardhome.init

```diff
@@ -22,11 +22,12 @@ start_service() {
 
   config_load adguardhome
   config_get WORK_DIR config workdir
+  config_get CONFIG_FILE config config
```

Comment:
Maybe better to use:
```
config_get CONFIG_FILE config config '/etc/adguardhome.yaml'
```
to ensure default value is set if that option is not present (like when upgrading existing package with existing config)?

Reply:
fixed